evm/vm/forks/spurious_dragon/transactions.py

```diff
@@ -32,6 +32,13 @@ def get_message_for_signing(self):
                 data=self.data,
             ))
 
+    def validate(self):
+        if is_eip_155_signed_transaction(self):
+            chain_id = extract_chain_id(self.v)
+            self.v_min = 35 + (2 * chain_id)
+            self.v_max = 36 + (2 * chain_id)
```

Comment:
Instead of assigning these to self, what do you think about making them computed properties that are instead derived on-demand.  That would remove the mutation and side-effects from this `validate` call.

Reply:
:+1: